docs/rule/link-rel-noopener.md

```diff
@@ -33,9 +33,7 @@ This rule **allows** the following:
 The following values are valid configuration:
 
 * string -- `strict` for enabled and validating both noopener `and` noreferrer
```

Comment:
Shouldn't we remove mention of `strict` since there's no option anymore?

Reply:
There is still an option for `strict` actually, it just has the same behavior as `true`.

Reply:
I was vaguely worried about folks "internalizing" that they had to use `"strict"` for this rule's configuration (since that has been true for a very long time in the 2.x series), and being confused when they swapped to `true`.

Reply:
I see. I'd personally rather do one of the following:
* remove the the `strict` option since it's redundant now
* or replace the `strict` option with a `lenient` option (default false) for people who want to opt back in to the old default behavior to ease migration

Reply:
updated